src/main/java/org/ow2/proactive/procci/request/CloudAutomationVariables.java

```diff
@@ -0,0 +1,114 @@
+package org.ow2.proactive.procci.request;
+
+import java.io.IOException;
+
+import org.ow2.proactive.procci.model.exception.CloudAutomationException;
+import lombok.AccessLevel;
+import lombok.Getter;
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpVersion;
+import org.apache.http.client.fluent.Request;
+import org.apache.http.client.fluent.Response;
+import org.apache.http.entity.ContentType;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * Created by mael on 06/10/16.
+ */
+
+/**
+ * Send crud request on cloud-automation-service/variables
+ */
+public class CloudAutomationVariables {
+
+    private static final Logger logger = LogManager.getRootLogger();
```

Comment:
Maybe it is better to initialize logger specific to CloudAutomationVariables.class, but not RootLogger.